ios/GoldWallet-beta.plist

```diff
@@ -57,8 +57,6 @@
 	<true/>
 	<key>NSAppTransportSecurity</key>
 	<dict>
-		<key>NSAllowsArbitraryLoads</key>
```

Comment:
will not impact network rulez in app like deeplinking or others secure ?

Reply:
Nope, it is enabled by default so not restrictions will be applied